lib/gutenberg/front-end.css

```diff
@@ -373,15 +373,13 @@ hr.wp-block-separator {
 		margin-right: calc(-100vw / 2 + 100% / 2);
 		width: 100vw;
 	}
-
```

Comment:
Not a huge deal, but we usually keep these spaces for CSS so spacing is even before and after the rule.

Reply:
hmm, that looks like a `stylelint` auto-fix

Reply:
So, in VS Code, when I added the extra space it was auto-tabbed to start writing more CSS and style lint complained about "No whitespace at EOL" or something like that. Simply removing the auto-tab prevent the auto-fix from messing things up.

:laughing: